src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -283,19 +280,26 @@ public Python(String n) {
       ServiceType st = sdt.get(i);
       // FIXME - cache in "data" dir Or perhaps it should be pulled into
       // resource directory during build time and packaged with jar
-      String url = String.format("https://raw.githubusercontent.com/MyRobotLab/pyrobotlab/%s/service/%s.py", p.getBranch(), st.getSimpleName());
+      String url = String.format("%s/%s.py", st.getSimpleName(), st.getSimpleName());

```

Comment:
I don't think this is a url anymore.. probably should rename the local variable to be filename ...

Reply:
sure.. agree .. changed ..